src/components/Grid/Grid.less

```diff
@@ -0,0 +1,66 @@
+.lucid-Grid {
+	.box-sizing();
+	display: flex;
+	flex-wrap: nowrap;
+
+	&-is-vertical {
+		flex-direction: column;
+	}
+	&-is-horizontal {
+		flex-direction: row;
+	}
+	&-is-multiline{
+		flex-wrap: wrap;
+	}
+	&-is-gutterless {
+		.lucid-Cell {
+			padding: 0;
+		}
+	}
+
+	.lucid-Grid { flex: 1 }
+}
+
+.lucid-Cell {
+	.box-sizing();
+	flex-basis: auto;
+	flex-grow: 1;
+	flex-shrink: 1;
+	padding: 10 0 0 (10/2);
+
+	&-is-full{
+		flex-grow: 0;
+		flex-shrink: 0;
+		flex-basis: 100%;
+	}
+	&-is-half{
+		flex-grow: 0;
```

Comment:
I read recently that the recommended way of writing flex is using the shorthand. So this would be `flex: 0 0 50%`. Not sure if it actually matters here or not.